### ISSUE TITLE
Aeronet's import_aeronet_data returning indexing error 

### ISSUE THREAD

**Issue by soypat** [robintw/Py6S#61]
Using a .dubovik file acquired using the step-by-step method  returns error. 

```
/usr/local/lib/python3.6/dist-packages/Py6S/SixSHelpers/aeronet.py in _get_aot(cls, df)
    206 
    207         # Remove the columns for AOT wavelengths with no data
--> 208         aot_df = df.ix[:, inds]
    209         aot_df = aot_df.dropna(axis=1, how='all')
    210         aot_df = aot_df.dropna(axis=0, how='any')

/usr/local/lib/python3.6/dist-packages/pandas/core/generic.py in __getattr__(self, name)
   5272             if self._info_axis._can_hold_identifiers_and_holds_name(name):
   5273                 return self[name]
-> 5274             return object.__getattribute__(self, name)
   5275 
   5276     def __setattr__(self, name: str, value) -> None:

AttributeError: 'DataFrame' object has no attribute 'ix'
```

A attribute/key check of the dataframe could be useful to spot errors early on and not raise them within the code.

**Comment by robintw** [comment]
Thanks - this is the same as #59, and I've just fixed it - will push commits and do a new release shortly.

**Comment by robintw** [comment]
Fixed in 7a6da6e.
